docs/start/framework/installation.md

```diff
@@ -23,11 +23,16 @@ You can now open your browser to `http://localhost:5173`
 
 You can [view the template on GitHub][default-template] to see how to manually set up your project.
 
-To get started with a template that deploys to your preferred host, check out [all of our templates](https://github.com/remix-run/react-router-templates).
+There are a number of [official templates][react-router-templates] available for you to get started with:
```

Comment:
```suggestion
There are a number of [official deployment templates][react-router-templates] available for you to get started with:
```

We should have the 'deployment' keyword in here. It's what people are looking for. And for some reason, the current content does not get found during search.

Reply:
Yeah that's fair, I tweaked it a bit. Really we just need to add a document on deploying, since `Installation` is not where people will look for deployment instructions